app/styles/_global.scss

```diff
@@ -206,3 +254,159 @@ fieldset[disabled] .form-control {
         width: 1170px;
     }
 }
+
+// Bootstrap Navbar
+
+.navbar-toggle {
+    position: relative;
+    float: right;
+    padding: 9px 10px;
+    margin-right: 15px;
+    margin-top: 8px;
+    margin-bottom: 8px;
+    background-color: transparent;
+    background-image: none;
+    border: 1px solid transparent;
+    border-radius: 4px;
+}
+
+.navbar-toggle:focus {
+    outline: 0;
+}
+
+.navbar-toggle .icon-bar {
+    display: block;
+    width: 22px;
+    height: 2px;
+    border-radius: 1px;
+}
+
+.navbar-toggle .icon-bar+.icon-bar {
+    margin-top: 4px;
+}
+
+.nav {
+    padding-left: 0;
+    margin-bottom: 0;
+    margin-top: 0;
```

Comment:
I'm not seeing this `margin-top` in the bundled css file. It doesn't seem to be impacting anything as far as I can tell, but curious

Reply:
This I recall adding, I forget exactly why. It may or may not still be necessary after making some of the more global changes.